test/commands/profiles/create.test.js

```diff
@@ -65,6 +76,36 @@ describe('commands', () => {
           );
         });
 
+      createTest([], { profileId: 'profile1', addProjects: 1 })
```

Comment:
Prefer `addProjects` be a list of project IDs to add. Otherwise, it's not obvious that this will add a project that matches the profileId.

Reply:
Thanks for the suggestion, Updated accordingly.